### DISCUSSION TITLE
cannot install

### DISCUSSION THREAD

**Discussion by a-my-n9** [flxzt/rnote#1273] [Q&A]
i am using win 11, surface pro 11, but i think win 11 should work? im not sure if i did anything wrong, tho. thanks
![image](https://github.com/user-attachments/assets/5fc239d1-e0ff-4416-8415-5159f8f2074c)

**Comment by Doublonmousse** [comment]
This is because we're not yet supporting windows on arm #837.
The installer also checks the architecture and stops the install if the architecture is not x64.

There's a PR opened, hopefully we can support windows on arm soon (with native compilation, not emulation)
